src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/ObjectModelConverters.cs

```diff
@@ -44,12 +65,10 @@ public static TestNode ToTestNode(this TestCase testCase, bool isTrxEnabled, ICl
     }
 
     private static void CopyVSTestProperties(IEnumerable<TestProperty> testProperties, TestNode testNode, TestCase testCase, Func<TestProperty, object?> getPropertyValue,
-        bool isTrxEnabled, IClientInfo client)
+        bool isTrxEnabled, bool addVSTestProviderProperties)
```

Comment:
Not like the idea of having `bool addVSTestProviderProperties` here. 

I would prefer to have 
```
if (addVstestProperties)
   CopyVSTestProperties(...);
```

rather than

```
CopyVSTestProperties(..., false);
```

Reply:
Same here.

`MSTestDiscoverer.TestCategory` needs to be copied regardless.